lxd/cluster/membership.go

```diff
@@ -198,8 +198,8 @@ func Accept(state *state.State, gateway *Gateway, name, address string, schema,
 	if err != nil {
 		return nil, errors.Wrap(err, "Failed to get raft nodes from the log")
 	}
-
-	if len(nodes) < membershipMaxRaftNodes {
+	count, err := Count(state)
```

Comment:
Please check the error, otherwise you end up ignoring it:

```go
if err != nil {
        return errors.Wrap(err, "Fetch cluster members count")
}
```

This should get rid of the static analysis failure reported by the test suite, look for:

```
/lxc-ci/build/tmp.blF58ZAwUA/go/src/github.com/lxc/lxd/lxd/cluster/membership.go:201:9: ineffectual assignment to err
```

in one of the logs, e.g. the [amd64/dir](https://jenkins.linuxcontainers.org/job/lxd-github-pull-test/7666/arch=amd64,backend=dir,compiler=golang-1.13,restrict=lxd-priv/console) one.